File: Ejercicio 8_Primer_Departamental.py
class Operaciones:

    def __init__(self):
        self.numeros = []

    def ingresar_numeros(self):
        for i in range(5):
            n = float(input("Ingresa un numero: "))
            self.numeros.append(n)

    def calcular_suma(self):
        return sum(self.numeros)

    def calcular_media(self):
        return self.calcular_suma() / len(self.numeros)

    def calcular_maximo(self):
        return max(self.numeros)

    def calcular_minimo(self):
        return min(self.numeros)

    def mostrar_resultados(self):
        print("Numeros ingresados:", self.numeros)
        print("Suma:", self.calcular_suma())
        print("Media:", self.calcular_media())
        print("Maximo:", self.calcular_maximo())
        print("Minimo:", self.calcular_minimo())


op = Operaciones()
op.ingresar_numeros()
op.mostrar_resultados()

File: Ejercicio 8_Primer_Departamental.cpp
#include <iostream>
#include <algorithm> 
using namespace std;

int main() {
    float n[5], suma = 0;

    for(int i = 0; i < 5; i++) {
        cout << "Ingrese el numero " << i + 1 << ": ";
        cin >> n[i];
        suma += n[i];
    }

    
    sort(n, n + 5);

    cout << "\n--- RESULTADOS ---" << endl;
    cout << "Suma: " << suma << endl;
    cout << "Promedio: " << suma / 5 << endl;
    cout << "Media (Mediana): " << n[2] << endl; 
    cout << "Maximo: " << n[4] << endl;
    cout << "Minimo: " << n[0] << endl;

    return 0;
}